### PR TITLE
Update the new-deck page

### PR DESCRIPTION
**Pull request by Vehmloewff**

### CODE REVIEW

**Comment by TehShrike** [comment]
Looks like a `.DS_Store` file got accidentally committed :-o

File: client/view/new-deck/NewDeck.html [hunk 1]
```diff
@@ -31,11 +31,14 @@ <h1>Add a new deck</h1>
...
 		<img srcset="images/google-spreadsheet-publish-example.png 2x" alt="A screenshot of the Google Sheets &quot;Publish to the Web&quot; dialog.">
+		<p>
+			Don't try to use the link that they give you here, it won't work.  It is very important that you share your spreadsheet first.
+		</p>
 		<p>
 			Close the "Publish to the Web" dialog.  Click the "Share" button at the top right.
 		</p>
 		<p>
-			Click "Get shareable link".  Then click the "Copy link" button.
+			Click "Get shareable link".
```
Comment (TehShrike) [inline client/view/new-deck/NewDeck.html:44]: Do you think it's clear enough without the "click the 'copy link'" message?

Comment (Vehmloewff) [reply client/view/new-deck/NewDeck.html:44]: When the user clicks "Get shareable link" it auto copies it to the clipboard.

File: client/view/new-deck/NewDeck.html [hunk 2]
```diff
@@ -31,11 +31,14 @@ <h1>Add a new deck</h1>
 			Click "Publish".
 		</p>
 		<img srcset="images/google-spreadsheet-publish-example.png 2x" alt="A screenshot of the Google Sheets &quot;Publish to the Web&quot; dialog.">
+		<p>
+			Don't try to use the link that they give you here, it won't work.  It is very important that you share your spreadsheet first.
```
Comment (TehShrike) [inline client/view/new-deck/NewDeck.html]:
```suggestion
			Don't try to use the link that they give you in the "Publish" window, it won't work.  It is very important that you share your spreadsheet first.
```

File: client/view/new-deck/NewDeck.html [hunk 3]
```diff
@@ -31,11 +31,14 @@ <h1>Add a new deck</h1>
...
+		</p>
 		<p>
 			Close the "Publish to the Web" dialog.  Click the "Share" button at the top right.
```
Comment (TehShrike) [inline client/view/new-deck/NewDeck.html]:
```suggestion
			Close the "Publish to the Web" dialog.
		</p>
		<p>
			Click the "Share" button at the top right.
```

**Comment by TehShrike** [comment]
Your "Remove .DS_Store" commit added a new `.DS_Store` file in `public/` 😂 

Other than a few other changes to the copy that I think would help, this looks good to me.

**Pull request merged**